e2e-tests/portal/cypress/e2e/journeys/admin/activate-deactivate-user/activate-deactivate-user.spec.js

```diff
@@ -12,7 +12,7 @@ context('Admin user updates an existing user', () => {
     firstname: 'first',
     surname: 'last',
     bank: 'Barclays Bank',
-    roles: ['maker'],
+    roles: ['admin', 'maker'],
```

Comment:
Should this be reverted? I'm not familiar with how `userToUpdate` is being used in this test exactly, but it reads like it's defining the user that an admin user will update 

Therefore should the test not be that a pure admin with only admin roles, updates a user that does _not_ have admin roles?

At the moment it looks like an admin user is updating another user that also has an admin role?

Reply:
Removed `admin`' from roles.